File: CS/SpreadWinDemoCS/chart/errorbar.cs
using FarPoint.Win.Chart;
using FarPoint.Win.Spread.Model;
using GrapeCity.Spreadsheet.Drawing;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SpreadWinDemo.chart
{
    public partial class errorbar : SpreadWinDemo.DemoBase
    {
        public errorbar()
        {
            InitializeComponent();

            // シートの設定
            InitSheet(fpSpread1.Sheets[0]);
        }

        private void InitSheet(FarPoint.Win.Spread.SheetView sheet)
        {
            fpSpread1.ActiveSheet.SetClip(0, 1, 1, 5, "1.2\t0\t-12.5\t-5\t15");
            fpSpread1.ActiveSheet.SetClip(1, 0, 1, 6, "1\t-15.43\t-11\t16\t0\t17.5");
            fpSpread1.ActiveSheet.SetClip(2, 0, 1, 6, "2\t7\t12\0\t-10\t10\t0");
            fpSpread1.ActiveSheet.AddChart(new CellRange(0, 0, 3, 6), typeof(FarPoint.Win.Chart.ClusteredBarSeries), 400, 250, 100, 100);
            if (fpSpread1.ActiveSheet.Charts[0].Model.PlotAreas[0].Series[0] is ClusteredBarSeries cluster)
            {
                foreach (BarSeries series in cluster.Series)
                {
                    // 誤差範囲の設定
                    ErrorBars errorBar = series.SetErrorBarsVisible(true);
                    errorBar.ValueType = ErrorBarValueType.StandardError;
                    errorBar.Type = FarPoint.Win.Chart.ErrorBarType.Both;
                }
            }
        }  
    }
}

File: VB/SpreadWinDemo/chart/errorbar.vb
Imports FarPoint.Win.Chart
Imports FarPoint.Win.Spread.Model

Public Class errorbar

    Public Sub New()

        InitializeComponent()

        ' シートの設定
        InitSheet(FpSpread1.Sheets(0))
    End Sub

    Private Sub InitSheet(sheet As FarPoint.Win.Spread.SheetView)
        FpSpread1.ActiveSheet.SetClip(0, 1, 1, 5, "1.2" & vbTab & "0" & vbTab & "-12.5" & vbTab & "-5" & vbTab & "15")
        FpSpread1.ActiveSheet.SetClip(1, 0, 1, 6, "1" & vbTab & "-15.43" & vbTab & "-11" & vbTab & "16" & vbTab & "0" & vbTab & "17.5")
        FpSpread1.ActiveSheet.SetClip(2, 0, 1, 6, "2" & vbTab & "7" & vbTab & "12" & vbTab & "0" & vbTab & "-10" & vbTab & "10" & vbTab & "0")
        FpSpread1.ActiveSheet.AddChart(New CellRange(0, 0, 3, 6), GetType(FarPoint.Win.Chart.ClusteredBarSeries), 400, 250, 100, 100)

        Dim chart = FpSpread1.ActiveSheet.Charts(0)
        Dim plotArea = chart.Model.PlotAreas(0)
        Dim cluster = TryCast(plotArea.Series(0), FarPoint.Win.Chart.ClusteredBarSeries)

        If cluster IsNot Nothing Then
            For Each series As FarPoint.Win.Chart.BarSeries In cluster.Series
                ' 誤差範囲の設定
                Dim errorBar As FarPoint.Win.Chart.ErrorBars = series.SetErrorBarsVisible(True)
                errorBar.ValueType = FarPoint.Win.Chart.ErrorBarValueType.StandardError
                errorBar.Type = FarPoint.Win.Chart.ErrorBarType.Both
            Next
        End If
    End Sub
End Class
